src/deployment/ios.md

```diff
@@ -161,21 +140,36 @@ app's icons:
 1. Verify the icon has been replaced by running your app using
    `flutter run`.
 
-## Create a build archive with Xcode
-
-This step covers creating a build archive and uploading
-your build to App Store Connect.
+## Create a build archive and upload to App Store Connect
 
 During development, you've been building, debugging, and testing
 with _debug_ builds. When you're ready to ship your app to users
 on the App Store or TestFlight, you need to prepare a _release_ build.
-At this point, you might consider [obfuscating your Dart code][]
```

Comment:
Moved to `Create an app bundle` section since that's when you would add the flags.